### PR TITLE
Resolve cache issue

### PR DESCRIPTION
**Pull request by RobStallion**
Resolve cache issue #315

### CODE REVIEW

**Comment by RobStallion** [comment]
This should solve the issue with the caching but if anyone/everyone could have a look to see if they can spot something I have missed it will be appreciated 👍

File: lib/cs_guide/postcode_lat_long.ex [hunk 1]
```diff
@@ -24,29 +23,82 @@ defmodule CsGuide.PostcodeLatLong do
         # against the cache. For testing at that point we can create a smaller
         # csv file which we can use to test with.
       _ ->
-        postcode_cache = :ets.new(:postcode_cache, [:set, :protected, :named_table])
-        store_postcodes_in_ets(postcode_cache)
+        :ets.new(:postcode_cache, [:set, :protected, :named_table])
```
Comment (nelsonic) [inline lib/cs_guide/postcode_lat_long.ex]: was variable assignment spiking the memory usage? 🤔 (_good catch!_)

Comment (RobStallion) [reply lib/cs_guide/postcode_lat_long.ex]: 😂 every bit counts. (I actually just removed it because it didn't really need to be assigned)

**Comment by SimonLab** [comment]
No more R14/R15 memory error on staging:
image

thanks @RobStallion :tada:

**Comment by nelsonic** [comment]
@SimonLab thanks for _confirming_. ✅

**Pull request merged**